.github/CODEOWNERS

```diff
@@ -41,6 +42,21 @@ browser/widevine @simonhong
 browser/ui/views/infobars/brave_wayback_machine_* @simonhong
 components/brave_wayback_machine @simonhong
 
+# WebUI
+/components/*_ui/ @petemill @cezaraugusto
+/components/brave_extension/ @cezaraugusto @petemill
+/components/brave_sync/extension/ @cezaraugusto
+/browser/resources/ @petemill
+/ui/webui/ @petemill
+/browser/ui/webui/ @petemill
+
+# Native UI
+/browser/ui/views/ @petemill
+/browser/ui/brave_actions/ @petemill
+/browser/ui/toolbar/ @petemill
+/browser/ui/brave_layout_constants.* @petemill
+/ui/base/
```

Comment:
Would you mind if I added myself to a few of these?
Specifically:
- /browser/resources/
- /ui/webui/
- /browser/ui/webui/

Reply:
@bsclifton of course it's up to you what you want to get automatic reviews on!